_locales/cs/maqueen-strings.json

```diff
@@ -0,0 +1,128 @@
+{
+  "Maqueen_V5.BatteryType.Alkaline|block": "alkalické",
+  "Maqueen_V5.BatteryType.Lithium|block": "lithiové",
+  "Maqueen_V5.BleCmd.BleBackward|block": "jeď dozadu",
+  "Maqueen_V5.BleCmd.BleForward|block": "jeď dopředu",
+  "Maqueen_V5.BleCmd.BleLeft|block": "zadboč vlevo",
+  "Maqueen_V5.BleCmd.BleRgbB|block": "RGB modrá",
+  "Maqueen_V5.BleCmd.BleRgbGB|block": "RGB azurová",
+  "Maqueen_V5.BleCmd.BleRgbG|block": "RGB zelený",
+  "Maqueen_V5.BleCmd.BleRgbOff|block": "RGB vypnuty",
+  "Maqueen_V5.BleCmd.BleRgbRB|block": "RGB fialová",
+  "Maqueen_V5.BleCmd.BleRgbRGB|block": "RGB bílá",
+  "Maqueen_V5.BleCmd.BleRgbRG|block": "RGB žlutá",
+  "Maqueen_V5.BleCmd.BleRgbR|block": "RGB červená",
+  "Maqueen_V5.BleCmd.BleRight|block": "zaboč vpravo",
+  "Maqueen_V5.BleCmd.BleServo1Left|block": "otoč servo1 doleva",
+  "Maqueen_V5.BleCmd.BleServo1Right|block": "otoč servo1 doprava",
+  "Maqueen_V5.BleCmd.BleServo2Left|block": "otoč servo2 doleva",
+  "Maqueen_V5.BleCmd.BleServo2Right|block": "otoč servo2 doprava",
+  "Maqueen_V5.BleCmdDefault|block": "proveď Bluetooth příkaz %cmd=bleCommand(jeď dopředu)",
+  "Maqueen_V5.BleGetCmd|block": "poslední Bluetooth příkaz",
+  "Maqueen_V5.BlecallbackUser|block": "při přijetí Bluetooth příkazu",
+  "Maqueen_V5.CarLightColors.Black|block": "černou",
+  "Maqueen_V5.CarLightColors.Blue|block": "modrou",
+  "Maqueen_V5.CarLightColors.Cyan|block": "azurovou",
+  "Maqueen_V5.CarLightColors.Green|block": "zelenou",
+  "Maqueen_V5.CarLightColors.Purple|block": "fialovou",
+  "Maqueen_V5.CarLightColors.Red|block": "červenou",
+  "Maqueen_V5.CarLightColors.White|block": "bílou",
+  "Maqueen_V5.CarLightColors.Yellow|block": "žlutou",
+  "Maqueen_V5.Dir.CCW|block": "vzad",
+  "Maqueen_V5.Dir.CW|block": "vpřed",
+  "Maqueen_V5.DirectionType.All|block": "na obou stranách",
+  "Maqueen_V5.DirectionType.Left|block": "vlevo",
+  "Maqueen_V5.DirectionType.Right|block": "vpravo",
+  "Maqueen_V5.I2CInit|block": "inicializuj I2C komunikaci s robotem",
+  "Maqueen_V5.Motors.All|block": "oba motory",
+  "Maqueen_V5.Motors.M1|block": "levý motor",
+  "Maqueen_V5.Motors.M2|block": "pravý motor",
+  "Maqueen_V5.neopixel_colors|block": "%color",
+  "Maqueen_V5.NeoPixelColors.Black|block": "černou",
+  "Maqueen_V5.NeoPixelColors.Blue|block": "modrou",
+  "Maqueen_V5.NeoPixelColors.Green|block": "zelenou",
+  "Maqueen_V5.NeoPixelColors.Indigo|block": "indigo",
+  "Maqueen_V5.NeoPixelColors.Orange|block": "oranžovou",
+  "Maqueen_V5.NeoPixelColors.Purple|block": "fialovou",
```

Comment:
I guess you accidentally swapped translations for Maqueen_V5.NeoPixelColors.Purple and Maqueen_V5.NeoPixelColors.Violet.

Violet = fialová (the rainbow colour)
Purple = purpurová OR (less common) nachová (as the redish/blueish/pinkish colour of clothes important / powerful people liked to wear in order to distinguish themselves).